pkg/watcher/internal/metricsprovider/signalfx.go

```diff
@@ -56,7 +56,9 @@ type signalFxClient struct {
 	client          http.Client
 	authToken       string
 	signalFxAddress string
+	hostKey			string
```

Comment:
Your editor has not replaced the tabs with spaces and these two lines appear misaligned